<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>701</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QWidget" name="page01"/>
    <widget class="QWidget" name="page02"/>
    <widget class="QWidget" name="page03"/>
    <widget class="QWidget" name="page04"/>
    <widget class="QWidget" name="page05"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuProductos">
    <property name="title">
     <string>Productos</string>
    </property>
    <addaction name="actionCategorias"/>
    <addaction name="actionProductos"/>
   </widget>
   <widget class="QMenu" name="menuVentas">
    <property name="title">
     <string>Ventas</string>
    </property>
    <addaction name="actionAgregar_Venta"/>
    <addaction name="actionMostrar_Ventas"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAyuda"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuProductos"/>
   <addaction name="menuVentas"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCategorias">
   <property name="text">
    <string>Categorias</string>
   </property>
  </action>
  <action name="actionProductos">
   <property name="text">
    <string>Productos</string>
   </property>
  </action>
  <action name="actionAgregar_Venta">
   <property name="text">
    <string>Agregar Venta</string>
   </property>
  </action>
  <action name="actionMostrar_Ventas">
   <property name="text">
    <string>Mostrar Ventas</string>
   </property>
  </action>
  <action name="actionAyuda">
   <property name="text">
    <string>Ayuda</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
